SU2_CFD/src/variables/CScalarVariable.cpp

```diff
@@ -59,4 +59,8 @@ CScalarVariable::CScalarVariable(unsigned long npoint, unsigned long ndim, unsig
   if (config->GetTime_Marching() == TIME_MARCHING::HARMONIC_BALANCE) {
     HB_Source.resize(nPoint, nVar) = su2double(0.0);
   }
+
+  /*--- Allocate residual structures ---*/
+
+  Res_TruncError.resize(nPoint,nVar) = su2double(0.0);
```

Comment:
Where are you using this?

Reply:
never ...  at the start I used the things over at feature_flamelet which meant copying a lot of unnecessary things . I'll remove it

Reply:
removed in be603aa